foundation-extensions/lifecycle-for-edge-network/README.md

```diff
@@ -1,42 +1,42 @@
 # Lifecycle for Edge Network
 
-The Adobe Experience Platform Mobile SDK Lifecycle extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/). The following steps will setup an application to forward Lifecycle extension metrics to the Adobe Experience Platform.
+The Adobe Experience Platform Mobile SDK Lifecycle extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/). Use the following steps to setup an application to forward Lifecycle extension metrics to the Adobe Experience Platform.
 
 
 ## Add AEP Mobile Lifecycle Details field group to your XDM Schema in Adobe Experience Platform
 
-Include the AEP Mobile Lifecycle Details field group to the schema defined in your Edge configuration data stream. The AEP Mobile Lifecycle field group defines all the metrics auto-collected by the Lifecycle for Edge Network extension. 
+Include the "AEP Mobile Lifecycle Details" field group to the schema defined in your Edge configuration data stream. The "AEP Mobile Lifecycle Details" field group defines all the metrics auto-collected by the Lifecycle for Edge Network extension. 
```

Comment:
Can we reword the following to avoid repetition? : "The "AEP Mobile Lifecycle Details" field group defines" -> "This field group defines ..."